cmd/gvproxy/main.go

```diff
@@ -56,12 +58,18 @@ func main() {
 	flag.StringVar(&vpnkitSocket, "listen-vpnkit", "", "VPNKit socket to be used by Hyperkit")
 	flag.StringVar(&qemuSocket, "listen-qemu", "", "Socket to be used by Qemu")
 	flag.StringVar(&bessSocket, "listen-bess", "", "unixpacket socket to be used by Bess-compatible applications")
+	flag.StringVar(&stdioSocket, "listen-stdio", "", "accept stdio pipe")
 	flag.Var(&forwardSocket, "forward-sock", "Forwards a unix socket to the guest virtual machine over SSH")
 	flag.Var(&forwardDest, "forward-dest", "Forwards a unix socket to the guest virtual machine over SSH")
 	flag.Var(&forwardUser, "forward-user", "SSH user to use for unix socket forward")
 	flag.Var(&forwardIdentify, "forward-identity", "Path to SSH identity key for forwarding")
 	flag.StringVar(&pidFile, "pid-file", "", "Generate a file with the PID in it")
 	flag.Parse()
+
+	if stdioSocket != "" {
+		log.SetOutput(os.Stderr)
```

Comment:
I believe this is generic and applies to all of the socket types. Perhaps better to put this behind a generic flag

Reply:
I removed this code since logrus defaults to stderr.